docs/podman-pod.1.md

```diff
@@ -0,0 +1,32 @@
+% podman-pod "1"
+
+## NAME
+podman\-pod - Simple management tool for groups of containers, called pods.
+
+## SYNOPSIS
+**podman pod** *subcommand*
+
+# DESCRIPTION
+podman pod is a set of subcommands that manage pods, or groups of containers.
+
+## SUBCOMMANDS
+
+| Subcommand                                        | Description                                                                    |
+| ------------------------------------------------- | ------------------------------------------------------------------------------ |
+| [podman-pod-create(1)](podman-pod-create.1.md)    | Create a new pod.                                                              |
+| [podman-pod-inspect(1)](podman-pod-inspect.1.md)  | Display a pod's configuration.                                                 |
```

Comment:
Can we remove these additional subcommand references and add them back in as we implement them?  To make that process more straightforward, we may want to use a definition-list like we do for options (although it looks like we have a fairly consistent precedent for using tables for subcommands).